.tekton/pull-request.yaml

```diff
@@ -49,7 +49,36 @@ spec:
           steps:
             - name: run-tests
               image: registry.access.redhat.com/ubi8/ubi:latest
-              script: cd $(workspaces.workspace.path) && ./automation/test.sh
+              workingDir: $(workspaces.workspace.path)
+              script: ./automation/test_prometheus.sh prepare
```

Comment:
1. Do you mind changing the name of the file to something more clear like just `promtool.sh`  ?

2. Eventually, I hope we can transform these shell scripts to something with an actual testing framework....

Reply:
To your second point, I think a Makefile would be nice improvement over shell scripts like this. Alternatively, building a container image with our testing tools preinstalled would eliminate most of it. We shouldn't use this PR for such a change but it's something to consider.

Reply:
alright